src/imitation/scripts/common/reward.py

```diff
@@ -84,17 +121,42 @@ def make_reward_net(
         add_std_alpha: multiple of reward function standard deviation to add to the
             reward in predict_processed. Must be None when using a reward function that
             does not keep track of variance. Defaults to None.
+        ensemble_size: The number of ensemble members to create. Must set if using
+            `net_cls =` :class: `reward_nets.RewardEnsemble`.
+        ensemble_member_config: The configuration for individual ensemble
+            members. Note that ensemble_members.net_cls must not be
+            :class: `reward_nets.RewardEnsemble`. Must be set if using
+            `net_cls = ` :class: `reward_nets.RewardEnsemble`.
 
     Returns:
         A, possibly wrapped, instance of `net_cls`.
+
+    Raises:
+        ValueError: Using a reward ensemble but fail to provide configuration.
     """
-    reward_net = reward_nets.make_reward_net(
-        venv.observation_space,
-        venv.action_space,
-        net_cls,
-        net_kwargs,
-        normalize_output_layer,
-        add_std_alpha,
-    )
-    logging.info(f"Reward network:\n {reward_net}")
-    return reward_net
+    if issubclass(net_cls, reward_nets.RewardEnsemble):
+        net_cls = typing.cast(Type[reward_nets.RewardEnsemble])
+        if ensemble_member_config is None:
+            raise ValueError("Must specify ensemble_member_config.")
+
+        if ensemble_size is None:
+            raise ValueError("Must specify ensemble_size.")
+
+        members = [
+            _make_reward_net(venv, **ensemble_member_config)
+            for _ in range(ensemble_size)
+        ]
+
+        reward_net = net_cls(venv.observation_space, venv.observation_space, members)
+
+        if add_std_alpha is not None:
+            reward_net = reward_nets.AddSTDRewardWrapper(
+                reward_net,
+                default_alpha=add_std_alpha,
+            )
+
+        # TODO should we support normalization on top of the ensemble or just within
```

Comment:
Good question. Given it'd complexify the code I lean towards not supporting it for now and we can add it later if a research use case emerges for it.